### DISCUSSION TITLE
How can I get direct video URL from tweet id?

### DISCUSSION THREAD

**Discussion by rushmedev** [redouane59/twittered#213] [Q&A]
For Example: https://twitter.com/Battlefield/status/**1404127027335155718**
For this tweet id: 1404127027335155718
I want to get raw video URL : https://video.twimg.com/amplify_video/1404068269137481735/vid/1280x720/CTESFJOA1NpNLqUa.mp4?tag=14

**Comment by redouane59** [comment]
Just tweet `https://twitter.com/Battlefield/status/1404127027335155718/video/1` and you will have the video directly

**Reply by rushmedev** [reply]
Hi @redouane59 thanks for the reply. Actually I want to get direct video link from the tweet so that so that I can extract the raw video and store it in AWS S3 storage. I am not sure if this feature is currently implemented yet. Please let me know.

**Reply by redouane59** [reply]
Hmmm do you know if that's something which currently exists in the Twitter API ?
